### DISCUSSION TITLE
【临时解决办法】手动设置“观看量”

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#1278] [其他]
绳网观看量怎么成114514力（悲
![image](https://github.com/user-attachments/assets/3b3c259e-6502-4a14-b2cb-5436b10bd3e3)

使用[MoeCount](https://count.kjchmc.cn/)手动添加观看量指示：
![测试](https://count.kjchmc.cn/get/@Test0f11cd2e?theme=asoul)

代码写法如下：
```![测试](https://count.kjchmc.cn/get/@Test0f11cd2e?theme=asoul)```
其中，```Test0f11cd2e```是需要计数的名字，根据自己帖子修改，但是不能和其他的重复，不然达不到计数的效果

**Comment by GanlinSama** [comment]
感谢大佬

**Comment by cpu644** [comment]
啊？

**Comment by capacitor1** [comment]
补充：Moecount有个缺点，就是每访问一次就增加1，即使只是测试。并且没办法重置。
